### ISSUE TITLE
Read .ass subtitles?

### ISSUE THREAD

**Issue by Urantij** [Aavtic/ass_parser#2]
I see examples, how you can create new subtitles or modify existing ones. But how can you read them? Access time, text, etc of events?  
Srt example shows .iter() method, but AssFile struct doesnt have it.

**Comment by Aavtic** [comment]
Hello @Urantij, Thank you for reporting this issue and bringing it to our attention!
I had the fields of the Event which has all the information regarding each Dialogue in the subtitle file as private, because I had only implemented APIs to create and modify the Subtitle files. 

The fix has been implemented in commit [f786cd8](https://github.com/Aavtic/ass_parser/commit/f786cd8996a8448f794170a8332b2459e9f79b61)

Now you can access each Dialogue field of Event by accessing the `.get_dialogues` method on the events.
You can refer this [example](https://github.com/Aavtic/ass_parser/blob/main/examples/read_existing_file.rs) to get a clear idea on how to view each fields of a Dialogue.

Example: 
``` rust
// Returns an iterator.
        let dialogues: Vec<Dialogue> = ass_file.events.get_dialogues();
        
            for dialogue in dialogues {
        println!("layer: {:?}", &dialogue.get_layer());
        println!("name: {:?}", &dialogue.get_name());
        println!("end: {:?}", &dialogue.get_end());
        println!("start: {:?}", &dialogue.get_start());
        println!("text: {:?}", &dialogue.get_text());
}     
```

I’m closing this issue for now, but feel free to reopen it if you experience the problem again. Thanks for your feedback!

**Comment by Aavtic** [comment]
A new version of `ass_parser` (0.2.1) has been published on crates.io you can update the crate using `cargo update` command and let me know if there are any other issues.

Kind regards
